src/mcts/params.cc

```diff
@@ -243,6 +246,7 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
   options->Add<IntOption>(kKLDGainAverageInterval, 1, 10000000) = 100;
   options->Add<FloatOption>(kMinimumKLDGainPerNode, 0.0f, 1.0f) = 0.0f;
+  options->Add<FloatOption>(kShortSightednessId, 0.0f, 1.0f) = 0.0f;
```

Comment:
Hehe.  Testing suggests values over 0.1 are pretty terrible - but I thought someone might like the comic value of the super aggressive play it would invite. (higher than 1 just felt silly :P)